File: C/for3.c
// for3.c
#include <stdio.h>

int main(void) {
  //
  for(int i = 1; i <= 9; i++) {
    for(int j = 1; j <= 9; j++) {
      printf("%d * %d = %2d\n", i, j, i * j);
    }
    printf("\n");
  }

  return 0;
}

File: Php/2026/EarlyPeriodTue34/for3.php
<?php  // for3.php

for ($i = 0; $i < 5; ++$i) {
    for ($j = 0; $j < 5; ++$j) {
        echo "* ";
    }
    echo "<br>";
}
echo "<br><br>";

for ($i = 0; $i < 5; ++$i) {
    for ($j = 0; $j <= $i; ++$j) {
        echo "* ";
    }
    echo "<br>";
}


for ($i = 0; $i < 5; ++$i) {
    $k = 5 - $i;
    for ($j = 0; $j < $k; ++$j) {
        echo "* ";
    }
    echo "<br>";
}